packages/indexer-agent/src/query-fees/allocations.ts

```diff
@@ -350,41 +351,56 @@ export class AllocationReceiptCollector implements ReceiptCollector {
     const hexPrefix = (bytes: string): string =>
       bytes.startsWith('0x') ? bytes : `0x${bytes}`
 
+    // TODO: investigate the cause of consistent failure
+    const numRetries = 1000
```

Comment:
Have you been seeing consistent failures in this function? 

Can we move this and below to another PR, so this one is focused more on the logs updates?